src/hlc_main.c

```diff
@@ -21,6 +21,10 @@
  */
 #include <hlc.h>
 
+#if defined(HL_MOBILE) && defined(sdl__Sdl__val)
```

Comment:
What's the point of having **sdl__Sdl__val** defined here ?

Reply:
It's to make sure that SDL is included before the hl.h, because we might eventually want to compile for mobile without SDL framework.